examples/templates/configuration/raw-keyrings/raw-aes-keyring.md

```diff
@@ -0,0 +1,132 @@
+[Example Templates](../../../examples.md#example-templates).
+
+## Implementations
+
+- [Python (DEV)](https://github.com/aws/aws-encryption-sdk-python/blob/keyring/examples/src/keyring/raw_aes/raw_aes.py)
+- [Python Master Key Provider (DEV)](https://github.com/aws/aws-encryption-sdk-python/blob/keyring/examples/src/master_key_provider/raw_aes/raw_aes.py)
+- [Java (DEV)](https://github.com/aws/aws-encryption-sdk-java/blob/keyring/src/examples/java/com/amazonaws/crypto/examples/keyring/rawaes/RawAes.java)
+- [Java Master Key Provider (DEV)](https://github.com/aws/aws-encryption-sdk-java/blob/keyring/src/examples/java/com/amazonaws/crypto/examples/masterkeyprovider/rawaes/RawAes.java)
+
+## Definitions
+
+### Conventions used in this document
+
+The key words
+"MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"
+in this document are to be interpreted as described in
+[RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Header
+
+```python
+# This examples shows how to configure and use a raw AES keyring.
+#
+# https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/choose-keyring.html#use-raw-aes-keyring
+#
+# In this example, we use the one-step encrypt and decrypt APIs.
+```
+
+## Summary
+
+```python
+# Demonstrate an encrypt/decrypt cycle using a raw AES keyring.
+```
+
+## Inputs
+
+- **source plaintext** :
+  Plaintext to encrypt
+
+## Steps
+
+1. Define encryption context.
+
+   ```python
+   # Prepare your encryption context.
+   # Remember that your encryption context is NOT SECRET.
+   # https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/concepts.html#encryption-context
+   encryption_context = {
+       "encryption": "context",
+       "is not": "secret",
+       "but adds": "useful metadata",
+       "that can help you": "be confident that",
+       "the data you are handling": "is what you think it is",
+   }
+   ```
+
+1. Generate the wrapping key.
+
+   ```python
+   # Generate a 256-bit (32 byte) AES key to use with your keyring.
+   #
+   # In practice, you should get this key from a secure key management system such as an HSM.
+   key = os.urandom(32)
+   ```
+
+1. Create keyring.
+
+   ```python
+   # Create the keyring that determines how your data keys are protected.
+   keyring = RawAESKeyring(
+       # The key namespace and key name are defined by you
+       # and are used by the raw AES keyring
+       # to determine whether it should attempt to decrypt
+       # an encrypted data key.
+       #
+       # https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/choose-keyring.html#use-raw-aes-keyring
+       key_namespace="some managed raw keys",
+       key_name="my AES wrapping key",
+       wrapping_key=key,
+   )
+   ```
+
+1. Encrypt plaintext data.
+
+   ```python
+   # Encrypt your plaintext data.
+   ciphertext, _encrypt_header = aws_encryption_sdk.encrypt(
```

Comment:
```suggestion
   ciphertext, encrypt_header = aws_encryption_sdk.encrypt(
```

I wasn't sure if the `_` was supposed to be there.

Reply:
It is, yes. In Python in this use, that denotes "I need to collect this variable and I want to give it a name for clarity, but I do not intend to ever use it."